src/__tests__/__snapshots__/typescript.js.snap

```diff
@@ -1,51 +1,41 @@
+"test/glamorous.test.tsx(20,15): error TS2345: Argument of type '{ ':hover': { '& .vendor-class': { height: string; }; }; }' is not assignable to parameter of type 'StyleArgument<CSSProperties, {}>'.
+  Type '{ ':hover': { '& .vendor-class': { height: string; }; }; }' is not assignable to type '(string | CSSProperties | StyleFunction<CSSProperties, {}>)[]'.
+    Property 'length' is missing in type '{ ':hover': { '& .vendor-class': { height: string; }; }; }'.
+test/glamorous.test.tsx(29,3): error TS2345: Argument of type '{ ':hover': { '& .vendor-class': { height: string; }; }; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}>'.
+  Object literal may only specify known properties, and '':hover'' does not exist in type 'StyleArgument<SVGProperties, {}>'.
+test/glamorous.test.tsx(52,3): error TS2345: Argument of type '{ overflowWrap: string; display: (\\"block\\" | \\"flex\\")[]; }' is not assignable to parameter of type 'StyleArgument<CSSProperties, {}>'.
+  Object literal may only specify known properties, and 'overflowWrap' does not exist in type 'StyleArgument<CSSProperties, {}>'.
+test/glamorous.test.tsx(57,3): error TS2345: Argument of type '{ textAlign: string; display: (\\"marker\\" | \\"block\\")[]; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}>'.
+  Object literal may only specify known properties, and 'textAlign' does not exist in type 'StyleArgument<SVGProperties, {}>'.
+test/glamorous.test.tsx(66,3): error TS2345: Argument of type '{ ':active': { \\"fontSize\\": number; }; '&:active': { \\"fontSize\\": number; \\"textAlign\\": string; }; }' is not assignable to parameter of type 'StyleArgument<CSSProperties, {}>'.
+  Object literal may only specify known properties, and ''&:active'' does not exist in type 'StyleArgument<CSSProperties, {}>'.
+test/should-fail.test.tsx(10,3): error TS2345: Argument of type '{ fillRule: \\"cat\\"; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}>'.
   Type '{ fillRule: \\"cat\\"; }' is not assignable to type '(string | SVGProperties | StyleFunction<SVGProperties, {}>)[]'.
     Property 'length' is missing in type '{ fillRule: \\"cat\\"; }'.
 test/should-fail.test.tsx(16,3): error TS2345: Argument of type '{ fillRule: \\"cat\\"; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}>'.
   Type '{ fillRule: \\"cat\\"; }' is not assignable to type '(string | SVGProperties | StyleFunction<SVGProperties, {}>)[]'.
     Property 'length' is missing in type '{ fillRule: \\"cat\\"; }'.
-test/should-fail.test.tsx(22,3): error TS2345: Argument of type '() => { fillRule: string; }' is not assignable to parameter of type 'StyleArgument<SVGProperties, {}>'.
-  Type '() => { fillRule: string; }' is not assignable to type '(string | SVGProperties | StyleFunction<SVGProperties, {}>)[]'.
-    Property 'push' is missing in type '() => { fillRule: string; }'.
```

Comment:
There are a lot of missing test failures in this file.

Looks like something has introduced a regression in the type safety.